.azure/publish-coverage.yml

```diff
@@ -29,11 +29,13 @@ pool:
 variables:
 - name: python-version
   value: 3.9
+- name: build_type
+  value: RelWithDebugInfo
```

Comment:
If this is supposed to be the CMake build type, shouldn't it be RelWithDebInfo?

Reply:
You know, it should be and I changed it, although the pipeline and tests didn't seem to care...